CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to the Reactive Streams Project
+
+The Reactive Streams project welcomes contributions from anybody who wants to participate in moving this initiative forward. All code or documentation that is contributed will have to be covered by a waiver of all copyrights and other rights as detailed by the LICENSE and COPYING files at each repository root, the rationale for this is that the APIs defined by this project shall be freely implementable and usable by everyone.
+
+## Code of Conduct
+
+This project is governed by the [Code of Conduct](CODE_OF_CONDUCT.md).
+By participating you are expected to uphold this code.
+
+## Copyright Statement
+
+The aforementioned waiver of copyrights and other rights is represented by the addition of a line to the file [CopyrightWaivers.txt](https://github.com/reactive-streams/reactive-streams-jvm/blob/master/CopyrightWaivers.txt). For a pull request to be considered every contributor must have signed the copyright statement in this way; this may be included within that same pull request.
```

Comment:
This needs to refer to the project-local CopyrightWaivers.txt

(currently refers to the JVM one) (https://github.com/reactive-streams/reactive-streams-jvm/blob/master/CopyrightWaivers.txt)